languages/java8/template/src/main/java/algorithmia/algorithm/AlgorithmBasic.java

```diff
@@ -0,0 +1,16 @@
+package algorithmia;
+import com.algorithmia.algorithmHandler.*;
+import com.algorithmia.*;
+
+class AlgorithmBasic
```

Comment:
Why are there two algorithms in the repo template?

Reply:
So we can decide as to which one we want to be the primary, and which we want to be secondary. I don't advocate that both should remain in the template when I merge this PR.